variables.tf

```diff
@@ -286,3 +291,21 @@ variable "maintenance_window" {
     update_track = "stable"
   }
 }
+
+# ----------------------------------------------------------------------------------------------------------------------
+# To enable Query Insights
+# ----------------------------------------------------------------------------------------------------------------------
+
+variable "insights_config" {
+  description = "The insights_config settings for the database."
+  type = object({
+    query_string_length     = number
+    record_application_tags = bool
+    record_client_address   = bool
+  })
+  default = {
+    query_string_length     = 1024
+    record_application_tags = false
+    record_client_address   = false
+  }
+}
```

Comment:
Kindly add a newline at the end of the file.

VSCode actually makes it easy by enabling an option to auto-add a newline at the end of every file.

Reply:
done